impress-org/givewp
# Fix: remove extra focus outline in block preview

Resolves: [GIVE-1189]

## Description
A change in styling in WP 6.61 is causing an extra outline to appear on focused blocks within our Visual Form Builder. This PR removes this outline to eliminate the double line visual.

## Affects
Visual Form Builder

## Visuals
### UI issue:
![double-border](https://github.com/user-attachments/assets/84f0c5e9-95e3-4098-a42d-4671f434af7e)

### UI fixed:
![remove-double-outline](https://github.com/user-attachments/assets/41929452-7eaa-4d8b-b23f-c7e1d3460b7e)

## Testing Instructions
- While using the latest version of WP edit or create a new form.
- Click on any block, you may need to click in between our block outline and the input.
- Verify that this additional outline is no longer visible.
- Ensure out block outline is still visible.
- We must test this on the latest version and the previous 2 versions of WP.

## Pre-review Checklist



[GIVE-1189]: https://stellarwp.atlassian.net/browse/GIVE-1189?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
